### PR TITLE
Add com.leinardi.gx52

### PR DESCRIPTION
**Pull request by leinardi**
I would like to publish my latest app on Flathub. More info on the app here: https://gitlab.com/leinardi/gx52

### CODE REVIEW

**Comment by leinardi** [comment]
bot, build com.leinardi.gx52

**Comment by leinardi** [comment]
bot, build com.leinardi.gx52

**Comment by leinardi** [comment]
bot, build com.leinardi.gx52

File: com.leinardi.gx52.json [hunk 1]
```diff
@@ -0,0 +1,119 @@
+{
+  "app-id": "com.leinardi.gx52",
+  "command": "gx52",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.36",
+  "sdk": "org.gnome.Sdk",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
```
Comment (bilelmoussaoui) [inline com.leinardi.gx52.json]: x11 only?

Comment (leinardi) [reply com.leinardi.gx52.json]: Oh right, I keep copying this from GWE which is X11 only and I forget about Wayland. Now should be fixed.

**Comment by leinardi** [comment]
bot, build com.leinardi.gx52

File: hidapi_fix_prefix.patch [hunk 2]
```diff
@@ -0,0 +1,20 @@
+--- a/setup.py	2020-02-07 23:32:40.000000000 +0100
```
Comment (bilelmoussaoui) [inline hidapi_fix_prefix.patch]: The patch doesn't seem to be used anywhere

Comment (leinardi) [reply hidapi_fix_prefix.patch]: Oh, good catch, it was a leftover. Removed.

**Comment by leinardi** [comment]
bot, build com.leinardi.gx52

**Comment by bilelmoussaoui** [comment]
I'm getting this
image
is that normal? otherwise looks good to me

**Comment by leinardi** [comment]
If you don't have a Logitech X52 connected should be normal. I'm currently working on showing a dialog explaining better that the hardware is missing or the udev rules are not there.

**Comment by bilelmoussaoui** [comment]
Right, looks good then

/merge

**Pull request closed without merging**